ActindoWorkFlow/Operation/04_CopyWorkflow.md

```diff
@@ -0,0 +1,102 @@
+
+
+## Export the workflow JSON code
+
+Export the JSON code of an existing workflow that you want to copy.
+
+#### Prerequisites
+
+- At least one workflow has been created, see [Create a workflow](./01_ManageWorkflows.md#create-a-workflow).
+- You have the required rights to edit a workflow.
+
+#### Procedure
+
+*Process Orchestration > Workflows > Tab OVERVIEW*
+
+![Workflows](../../Assets/Screenshots/ActindoWorkFlow/Workflows/Workflows.png "[Workflows]")
+
+1. Click the workflow you want to copy in the list of workflows.   
+    The *Workflow versions* view with all versions of the selected workflow is displayed.
+
+    ![Workflow versions](../../Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowVersions.png "[Workflow versions]")
+
+2. Click the workflow version you want to copy in the list of workflow versions.  
+    The workflow editor is displayed in the workspace.
+
+    ![Workflow editor](../../Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowEditor.png "[Workflow editor]")
+
+[comment]: <> (Step 2 as well as the screenshot workflow versions is possibly not needed in the next workflows version as the version view will disappear and clicking the workflow will take you to the editor directly, check next time > version view is still visible 21.02.2024)
+
+3. Click the ![Points](../../Assets/Icons/Points02.png "[Points]") (Points) button in the upper left corner next to the workflow name.   
+    The workflow context menu is displayed.
+
+    ![Context menu](../../Assets/Screenshots/ActindoWorkFlow/Workflows/ContextMenu.png "[Context menu]")
+
+4. Click the *Export JSON ..* menu entry in the context menu.   
+    The *Export JSON* window with the JSON code for the current workflow is displayed.
+
+    ![Export JSON](../../Assets/Screenshots/ActindoWorkFlow/Workflows/ExportJSON.png "[Export JSON]")
+
+5. Select the complete JSON code in the window, copy it to the clipboard and click the [CLOSE] button in the bottom right corner.   
+    The *Export JSON* window is closed.
+
+6. Click the ![Back](../../Assets/Icons/Back02.png "[Back]") (Back) button in the workflow editor.  
+    The workflow editor is closed and the list of workflows is displayed again.
+
+
+## Import the workflow JSON code
+
+Import the exported JSON code to an existing or new workflow to copy it.
+
+#### Prerequisites
+
+- At least one workflow has been created, see [Create a workflow](./01_ManageWorkflows.md#create-a-workflow).
+- You have the required rights to edit a workflow.
+- The JSON code of the workflow to copy has been exported to the clipboard, see [Export the workflow JSON code](#export-the-workflow-json-code).
+
+#### Procedure
+
+*Process Orchestration > Workflows > Tab OVERVIEW*
+
+![Workflows](../../Assets/Screenshots/ActindoWorkFlow/Workflows/Workflows.png "[Workflows]")
+
+1. [Create a new workflow](./01_ManageWorkflows.md#create-a-workflow) or click the workflow in the list of workflows to which you want to import the copied JSON code.   
```

Comment:
[Create a new workflow](./01_ManageWorkflows.md#create-a-workflow) or -> Das passt nicht ganz mit dem Prerequisite oben (At least one workflow has been created...). Ich verstehe, was du damit meinst. Vielleicht als Info dazu, oder einfach einen Satz dazu schreiben? Alternatively, you can also create a new workflow...